src/pages/instances/InstanceOverviewDevices.tsx

```diff
@@ -52,6 +64,53 @@ const InstanceOverviewDevices: FC<Props> = ({ instance }) => {
           role: "rowheader",
           "aria-label": "Type",
         },
+        {
+          content: (
```

Comment:
let's move this into a dedicated function to generate the detail column component. I find it difficult to read to have JSX directly embedded in JS code. So we can for example, have a function `getDeviceDetail` that takes in a device, then return some component.

Edit: actually, I think it would be easier to generate the entire detail column object, it would be easier for variable reuse for other column attributes aside from `content`